uberfire-api/src/main/java/org/uberfire/mvp/BiParameterizedCommand.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.mvp;
+
+@FunctionalInterface
+public interface BiParameterizedCommand<T, U> {
```

Comment:
Docs ->

Paulo, this is not a bifunction in order to keep the relation with ParameterizedCommand<t> right? Maybe relate this with bifunction java interface just to help us understanding. Also, could you please refer this class ParametrizedCommand?

Reply:
Docs done. Also referred to it in `ParameterizedCommand`.